web/src/main/kotlin/no/nav/su/se/bakover/web/Extensions.kt

```diff
@@ -155,9 +155,13 @@ internal suspend fun ApplicationCall.withKlageId(ifRight: suspend (UUID) -> Unit
 }
 
 internal suspend inline fun <reified T> ApplicationCall.withBody(ifRight: (T) -> Unit) {
-    Either.catch { deserialize<T>(this) }.fold(
+    val body = this.receiveTextUTF8()
```

Comment:
Burde denne også være innenfor `Either.catch`? I tilfelle man ikke får datastreamen

Reply:
Det er jo aspektet med sikkerlogg da. Hmm :thinking:

Reply:
Jeg tenkte mer med tanke på `this.svar` -- hvis `receiveTextUTF8` kaster en feil blir vel det ikke noen respons til klienten? Eller har vi en ytre implisitt try-catch som fanger opp og gir en 500 f.eks.?

Reply:
Vi har en ytre implisitt try-catch ja som gir 500 med json-body ukjent_feil